### PR TITLE
[cli] Add auto renew prompt to domains buy

### PR DESCRIPTION
**Pull request by mglagola**
Add auto renew prompt to `domains buy`

```diff
 > The domain "example.com" is available to buy under <username>!
 > Buy now for $20 (1yr)? [y|N] 
+ > Auto renew yearly for $20? [Y|n] 
```

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by TooTallNate — DISMISSED** [review]

**Comment by TooTallNate** [comment]
Seems the test didn't like the change.

**Review by anatrajkovska — DISMISSED** [review]

**Review by styfle — DISMISSED** [review]

**Review by TooTallNate — DISMISSED** [review]

File: packages/now-cli/test/integration.js [hunk 1]
```diff
@@ -1335,12 +1336,19 @@ test('try to purchase a domain', async t => {
     return;
   }
 
+  const stream = new Writable();
+  stream.pipe = dest => {
+    dest.write('y');
+    dest.write('y');
+    return dest;
+  };
```
Comment (TooTallNate) [inline packages/now-cli/test/integration.js]: Try like this:

```suggestion
stream.push('y\n');
stream.push('y\n');
stream.push(null);
```

File: packages/now-cli/test/integration.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import path from 'path';
 import { URL, parse as parseUrl } from 'url';
 import test from 'ava';
 import semVer from 'semver';
+import { Writable } from 'stream';
```
Comment (mglagola) [inline packages/now-cli/test/integration.js]:
```suggestion
import { Readable } from 'stream';
```

File: packages/now-cli/test/integration.js [hunk 3]
```diff
@@ -1335,12 +1336,19 @@ test('try to purchase a domain', async t => {
     return;
   }
 
+  const stream = new Writable();
```
Comment (mglagola) [inline packages/now-cli/test/integration.js]:
```suggestion
  const stream = new Readable();
```

**Review by styfle — DISMISSED** [review]

**Review by styfle — APPROVED** [review]

**Review by anatrajkovska — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**